### ISSUE TITLE
Api changed in 0.11

### ISSUE THREAD

**Issue by janLo** [happyleavesaoc/python-snapcast#3]
The clients that are returned from Server.GetStatus are now organized in "groups". Therefore the lib fail:
```
Traceback (most recent call last):
  File "/home/hass/hass_env/lib/python3.4/site-packages/homeassistant/helpers/entity_component.py", line 153, in _async_setup_platform
    entity_platform.schedule_add_entities, discovery_info
  File "/usr/lib/python3.4/asyncio/futures.py", line 388, in __iter__
    yield self  # This tells Task to wait for completion.
  File "/usr/lib/python3.4/asyncio/tasks.py", line 286, in _wakeup
    value = future.result()
  File "/usr/lib/python3.4/asyncio/futures.py", line 277, in result
    raise self._exception
  File "/usr/lib/python3.4/concurrent/futures/thread.py", line 54, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/hass/hass_env/lib/python3.4/site-packages/homeassistant/components/media_player/snapcast.py", line 43, in setup_platform
    server = snapcast.control.Snapserver(host, port)
  File "/home/hass/hass_env/lib/python3.4/site-packages/snapcast/control.py", line 167, in __init__
    self.synchronize()
  File "/home/hass/hass_env/lib/python3.4/site-packages/snapcast/control.py", line 242, in synchronize
    for client in status.get('clients'):
TypeError: 'NoneType' object is not iterable

```

**Comment by pbros** [comment]
I am also interested in a version that supports v0.11. Thank you!

**Comment by janLo** [comment]
@happyleavesaoc , it seems you've put some work into supporting 0.11. Can you tell what is implemented, what not and, if so, what outside api has changed? I would like to upgrade the component in https://github.com/home-assistant/home-assistant

**Comment by happyleavesaoc** [comment]
@janLo Everything is supported at this point. I will be updating the documentation to reflect a few small API changes.

I also have an updated Home Assistant component prepared already!

I expect everything to be ready in a day or so.

**Comment by happyleavesaoc** [comment]
`2.0.0` has been released, with support for Snapcast `0.11.1`.
